### ISSUE TITLE
Regression that prevents setting the WakeAllowed property

### ISSUE THREAD

**Issue by RyuzakiKK** [bluez/bluez#1045]
## Steps to reproduce

- Set `KernelExperimental = 15c0a148-c273-11ea-b3de-0242ac130004` in `main.conf`
  - aka "LL Privacy"
- Compile and install Bluez 5.79
- Pair a device that supports wake from suspend, e.g. the Xbox Series controller
- Toggle the WakeAllowed property
  ```
  $ dbus-send --print-reply=literal --system --dest=org.bluez /org/bluez/hci0/dev_3C_FA_06_46_53_ED \
  org.freedesktop.DBus.Properties.Set string:org.bluez.Device1 string:WakeAllowed "variant:boolean:false"
  ```

## Expected result

The `dbus-send` command should nearly immediately return and the WakeAllowed property for that device should be set accordingly.

## Actual result

The `dbus-send` command fails with:
```
Error org.freedesktop.DBus.Error.NoReply: Did not receive a reply. Possible causes include: the remote
application did not send a reply, the message bus security policy blocked the reply, the reply timeout expired,
or the network connection was broken.
```

This is the btmon when running the `dbus-send` command: [9cc5879.zip](https://github.com/user-attachments/files/18165959/9cc5879.zip)

The `Set Device Flags` command seems to be completely missing?

## Additional info

With a bisect I was able to track down the regression to https://github.com/bluez/bluez/commit/9cc587947b6ac56a4c94dcc880b273bc72af22a8.

After compiling `7a45038`, the parent commit of `9cc5879`, I used btmon again:  [7a45038dc.zip](https://github.com/user-attachments/files/18166042/7a45038dc.zip)

When setting the property, this time we get the expected `Set Device Flags` command.

**Comment by RyuzakiKK** [comment]
@Vudentz I confirm that the proposed patch above is fixing the issue. Thanks!

**Comment by RyuzakiKK** [comment]
@Vudentz I'm sorry, I continued to test the patch that you provided, and unfortunately from time to time I can still trigger the "Error org.freedesktop.DBus.Error.NoReply" or "Error org.bluez.Error.Busy: Property change in progress".

It seems the repro to be:
- Pair a device
- Keep the device connected
- Try to set the WakeAllowed property

The problem is that it doesn't seem to be a 100% reproducer. So far I was able to trigger it with both an Xbox Series X|S controller and a Dualsense controller.

I also noticed that setting the WakeAllowed property with the device disconnected seems to always work. And after doing that once, it seems to unblock even setting the property when the device is connected.

If that's useful, I can try to grab the logs with btmon.
